flight/Modules/VtolPathFollower/vtol_follower_fsm.c

```diff
@@ -641,12 +639,17 @@ static void go_enable_rise_here()
 
 	float down = positionActual.Down;
 
+	float tmp;
```

Comment:
Can you name this `rth_alt` or similar? It'll make the code below a bit more readable.

Reply:
I can split it into two named vars

Reply:
Ahhh, yeah, didn't spot that it was used for multiple things. I think splitting is best tbh. Compiler should be able to figure out it can use the same register.